plugins/embed-optimizer/load.php

```diff
@@ -2,7 +2,7 @@
 /**
  * Plugin Name: Embed Optimizer
  * Plugin URI: https://github.com/WordPress/performance/tree/trunk/plugins/embed-optimizer
- * Description: Optimize the performance of embeds.
+ * Description: Optimize the performance of embeds by lazy-loading them.
```

Comment:
https://github.com/WordPress/performance/issues/1046#issuecomment-2005134234 proposes `Optimizes the performance of embeds by lazy-loading iframes and scripts.`.

Doesn't mean we have to go with that, but let's align this. At the very minimum, this needs to be changed to use a third-person verb.